fabtests/include/shared.h

```diff
@@ -118,6 +118,8 @@ enum {
 	FT_OPT_SKIP_MSG_ALLOC	= 1 << 12,
 	FT_OPT_SKIP_REG_MR	= 1 << 13,
 	FT_OPT_OOB_ADDR_EXCH	= 1 << 14,
+	FT_OPT_ALLOC_MULT_TX	= 1 << 15,
+	FT_OPT_ALLOC_MULT_RX	= 1 << 16,
```

Comment:
I would combine these into a single option, unless they really need to be separate.